### PR TITLE
created contact listing page UI

### LINKED ISSUE #10: 4- Create List Contacts UI

### PR DESCRIPTION
**Pull request by shivamkrs89**

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/MenuBar.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+
+import { Search, Settings, Plus } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
+
+const ContactsMenuBar = ({ showMenu }) => {
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/MenuBar.jsx]: Go through the Slack thread very carefully. https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663590973475389?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

File: app/javascript/src/components/Dashboard/Contacts/MenuBar.jsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  const [isSegmentsSearchCollapsed, setIsSegmentsSearchCollapsed] =
+    useState(true);
+  const [isTagsSearchCollapsed, setIsTagsSearchCollapsed] = useState(true);
+
+  return (
+    <div className="flex">
+      <MenuBar showMenu={showMenu} title="Notes">
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/MenuBar.jsx]: Why do we need this `div` here? Why not pass in these classNames as part of the MenuBar itself?

File: app/javascript/src/components/Dashboard/Contacts/Table/TableColumn.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import { buildContactsTableColumnData } from "./utils";
+
+export const CONTACTS_TABLE_COLUMNS = buildContactsTableColumnData();
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Table/TableColumn.js]: You are not paying attention to the filenames. Go through the Slack thread.

File: app/javascript/src/components/Dashboard/Contacts/Table/utils.jsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    width: "30%",
+    key: "name",
+    dataIndex: "name",
+    render: (name, { role, imageUrl }) => (
+      <div className="flex space-x-3">
+        <Avatar size="large" user={{ imageUrl, name }} />
+        <div>
+          <Typography style="h4">{name}</Typography>
+          <Typography style="body2">{role}</Typography>
+        </div>
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Table/utils.jsx]: Why do we need to use a `div` here over a React Fragment?

**Review by yedhink — CHANGES_REQUESTED** [review]
@shivamkrs89

File: app/javascript/src/components/Dashboard/Contacts/Table/TableColumn.js (hunk 3 above)
Comment (shivamkrs89) [reply app/javascript/src/components/Dashboard/Contacts/Table/TableColumn.js]: changed the file name to column.js and pushed the changes.

File: app/javascript/src/components/Dashboard/Contacts/MenuBar.jsx (hunk 1 above)
Comment (shivamkrs89) [reply app/javascript/src/components/Dashboard/Contacts/MenuBar.jsx]: changed the component and filename both to SideMenu.jsx and pushed the changes.

File: app/javascript/src/components/Dashboard/Contacts/Table/column.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+import { buildContactsTableColumnData } from "./utils";
+
+export const CONTACTS_TABLE_COLUMN = buildContactsTableColumnData();
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Table/column.js]: https://www.loom.com/share/01c7bf8017a9473b9539604b0c3f6ba0

**Review by yedhink — CHANGES_REQUESTED** [review]
@shivamkrs89

File: app/javascript/src/components/Dashboard/Contacts/Table/column.js (hunk 5 above)
Comment (shivamkrs89) [reply app/javascript/src/components/Dashboard/Contacts/Table/column.js]: Ok Yedhin, I have directly imported 
`import { buildContactsTableColumnData } from "./utils";`  and done required changes in **index.js** file as:
`columnData={buildContactsTableColumnData()}`
 And deleted **column.js** file.
Thanks.

**Pull request merged**